dmotion:
  com_x:
    data: [0, 0.96676599318442413, 1.3954172521278483, 1.9929880323659088, 2.9197368882652173, 3.7500000000000053, 5.5, 1.4999999999999996, 1.4991584774626858, 0.7399661350210486, 1.19919451209784236, 1.7868755055511703, 1.6999999999999996, 1.6999999999999996, 0.0001, -0.0001, 0, -0.061262994851012643, 0.54295569963843737, -0.0033906714980246624, -9.9999999999999815e-05]
